### PR TITLE
Update ProfileId's local claim URI

### PR DESCRIPTION
**Pull request by asha15**
## Purpose
This PR fixes https://github.com/wso2/product-is/issues/13785

### CODE REVIEW

File: en/docs/learn/configuring-outbound-provisioning.md [hunk 1]
```diff
@@ -91,12 +91,14 @@ Pickup now needs to set up outbound provisioning between WSO2 IS and the **Sales
     | LanguageLocaleKey                  | http://wso2.org/claims/dob                      |
     | LastName                           | http://wso2.org/claims/lastname                 |
     | LocaleSidKey                       | http://wso2.org/claims/primaryChallengeQuestion |
-    | ProfileId                          | http://wso2.org/claims/role                     |
+    | ProfileId                          | None                     |
```
Comment (piraveena) [inline en/docs/learn/configuring-outbound-provisioning.md:94]: Do we need this None mapping

File: en/docs/learn/outbound-provisioning-with-salesforce.md [hunk 2]
```diff
@@ -281,14 +281,14 @@ Provider.
     | LanguageLocaleKey                  | `               http://wso2.org/claims/dob              `                      |
     | LastName                           | `               http://wso2.org/claims/lastname              `                 |
     | LocaleSidKey                       | `               http://wso2.org/claims/primaryChallengeQuestion              ` |
-    | ProfileId                          | `               http://wso2.org/claims/role              `                     |
+    | ProfileId                          | `               None             `                                             |
```
Comment (piraveena) [inline en/docs/learn/outbound-provisioning-with-salesforce.md:284]: Do we need this None mapping

File: en/docs/learn/outbound-provisioning-with-salesforce.md [hunk 3]
```diff
@@ -281,14 +281,14 @@ Provider.
...
     | TimeZoneSidKey                     | `               http://wso2.org/claims/challengeQuestion1              `       |
     | UserPermissionsCallCenterAutoLogin | `               http://wso2.org/claims/telephone              `                |
     | UserPermissionsMarketingUser       | `               http://wso2.org/claims/mobile              `                   |
     | UserPermissionsOfflineUser         | `               http://wso2.org/claims/country              `                  |
     | Username                           | `               http://wso2.org/claims/emailaddress              `             |
-
-    ![add-claim-mapping](../assets/img/using-wso2-identity-server/add-claim-mapping.png) 
+    
+    ![add-claim-mapping](https://user-images.githubusercontent.com/28682701/174729411-c9c201b1-a376-4c73-8d61-cd230858a923.png)
```
Comment (piraveena) [inline en/docs/learn/outbound-provisioning-with-salesforce.md:291]: Images need to be added into the doc-is repo and pointed from the md file

File: en/docs/learn/configuring-outbound-provisioning.md (hunk 1 above)
Comment (asha15) [reply en/docs/learn/configuring-outbound-provisioning.md:94]: yes, we should not map any local claim to profileId. we should keep it blank

File: en/docs/learn/outbound-provisioning-with-salesforce.md (hunk 2 above)
Comment (asha15) [reply en/docs/learn/outbound-provisioning-with-salesforce.md:284]: please check the previous review's reply

File: en/docs/learn/outbound-provisioning-with-salesforce.md (hunk 3 above)
Comment (asha15) [reply en/docs/learn/outbound-provisioning-with-salesforce.md]: Did the changes. Thank you!

File: en/docs/learn/configuring-outbound-provisioning.md (hunk 1 above)
Comment (piraveena) [reply en/docs/learn/configuring-outbound-provisioning.md:94]: then shall we remove the mapping

File: en/docs/learn/outbound-provisioning-with-salesforce.md (hunk 2 above)
Comment (piraveena) [reply en/docs/learn/outbound-provisioning-with-salesforce.md:284]: then shall we remove the mapping

File: en/docs/learn/configuring-outbound-provisioning.md (hunk 1 above)
Comment (asha15) [reply en/docs/learn/configuring-outbound-provisioning.md:94]: If we remove the mapping we can't add the ProfileId in `Advanced Claim Configuration`.

**Review by piraveena — APPROVED** [review]

File: en/docs/learn/outbound-provisioning-with-salesforce.md (hunk 2 above)
Comment (asha15) [reply en/docs/learn/outbound-provisioning-with-salesforce.md:284]: If we remove the mapping we can't add the ProfileId in Advanced Claim Configuration, so the API call will fail.

Comment (nilminiwso2) [reply en/docs/learn/outbound-provisioning-with-salesforce.md:284]: This give the impression that `None` is the value to add here.
Therefore, shall we say the following:

```
| ProfileId                          | Empty                                           |
```

Comment (asha15) [reply en/docs/learn/outbound-provisioning-with-salesforce.md:284]: yeah that's better. will do the changes. Thank you!

Comment (asha15) [reply en/docs/learn/outbound-provisioning-with-salesforce.md]: @nilminiwso2 did the changes. please review

File: en/docs/learn/outbound-provisioning-with-salesforce.md [hunk 4]
```diff
@@ -281,14 +281,14 @@ Provider.
     | LanguageLocaleKey                  | `               http://wso2.org/claims/dob              `                      |
     | LastName                           | `               http://wso2.org/claims/lastname              `                 |
     | LocaleSidKey                       | `               http://wso2.org/claims/primaryChallengeQuestion              ` |
-    | ProfileId                          | `               http://wso2.org/claims/role              `                     |
+    | ProfileId                          | `               Empty            `                                             |
```
Comment (nilminiwso2) [inline en/docs/learn/outbound-provisioning-with-salesforce.md:284]:
```suggestion
    | ProfileId                          | Empty                                             |
```

**Pull request merged**